tests/gtest/test_fims_math_ad_fabs_min_max.cpp

```diff
@@ -0,0 +1,46 @@
+#include "gtest/gtest.h"
+#include "common/fims_math.hpp"
+
+namespace
+{
+
+  // Test ad_fabs
+  TEST(ad_fabs, use_double_values)
+  {
+    // Expected value from R: x=2; sqrt(x*x+1e-5) = 2.000002
+    EXPECT_NEAR(fims::ad_fabs(2.0), 2.000002, 0.000001);
+    EXPECT_NE(fims::ad_fabs(2.0), 1.0);
```

Comment:
should this be `EXPECT_NEAR` or is `EXPECT_NE` correct?

Reply:
I just read some documentation and realized that `EXPECT_NE` means expect not equal

Reply:
You are correct!